aidants_connect_habilitation/tests/test_functionnal/test_personnel_form.py

```diff
@@ -114,6 +132,266 @@ def test_js_added_aidant_form_has_correct_id(self):
+            issuer=issuer,
+            manager=manager,
+            data_privacy_officer=dpo,
+        )
+
+        self.__open_form_url(issuer, organisation)
+
+        self.selenium.find_element(By.CSS_SELECTOR, '[type="submit"]').click()
+
+        path = reverse(
+            "habilitation_validation",
+            kwargs={
+                "issuer_id": str(organisation.issuer.issuer_id),
+                "draft_id": str(organisation.draft_id),
+            },
+        )
+
+        WebDriverWait(self.selenium, 10).until(url_matches(f"^.+{path}$"))
+
+        organisation.refresh_from_db()
+        self.assertEqual(organisation.aidant_requests.count(), 0)
+
+    def test_form_submit_multiple_aidants(self):
+        issuer: Issuer = IssuerFactory()
+        dpo: DataPrivacyOfficer = DataPrivacyOfficerFactory()
+        manager: Manager = ManagerFactory()
+        organisation: OrganisationRequest = DraftOrganisationRequestFactory(
+            issuer=issuer,
+            manager=manager,
+            data_privacy_officer=dpo,
+        )
+
+        # Setup 2 initial requests
+        AidantRequestFactory(organisation=organisation)
+        AidantRequestFactory(organisation=organisation)
+
+        self.__open_form_url(issuer, organisation)
+
+        for i in range(2, 6):
+            aidant_form: AidantRequestForm = get_form(AidantRequestForm)
+            aidant_data = aidant_form.cleaned_data
+            for field_name in aidant_form.fields:
+                element: WebElement = self.selenium.find_element(
+                    By.CSS_SELECTOR,
+                    f"#id_{PersonnelForm.AIDANTS_FORMSET_PREFIX}-{i}-{field_name}",
+                )
+                element.clear()
+                element.send_keys(aidant_data[field_name])
+
+            self.selenium.find_element(By.CSS_SELECTOR, "#add-aidant-btn").click()
+
+        self.selenium.find_element(By.CSS_SELECTOR, '[type="submit"]').click()
+
+        path = reverse(
+            "habilitation_validation",
+            kwargs={
+                "issuer_id": str(organisation.issuer.issuer_id),
+                "draft_id": str(organisation.draft_id),
+            },
+        )
+
+        WebDriverWait(self.selenium, 10).until(url_matches(f"^.+{path}$"))
+
+        organisation.refresh_from_db()
+        self.assertEqual(organisation.aidant_requests.count(), 6)
+
+    def test_form_submit_modify_multiple_aidants(self):
+        issuer: Issuer = IssuerFactory()
+        dpo: DataPrivacyOfficer = DataPrivacyOfficerFactory()
+        manager: Manager = ManagerFactory()
+        organisation: OrganisationRequest = DraftOrganisationRequestFactory(
+            issuer=issuer,
+            manager=manager,
+            data_privacy_officer=dpo,
+        )
+
+        AidantRequestFactory(organisation=organisation)
+        AidantRequestFactory(organisation=organisation)
+        AidantRequestFactory(organisation=organisation)
+        AidantRequestFactory(organisation=organisation)
```

Comment:
```suggestion
        for _ in range(4):
            AidantRequestFactory(organisation=organisation)
```

Reply:
(Petite pensée pour ce jour où j'ai mis une boucle sans fin dans un test en écrivant `for _ in itertools.count(4):`. Heureusement j'ai pas push. ^____^

Reply:
Je n'ai pas bien compris pourquoi tu génères 4 aidants côté BDD par contre ?

Reply:
AH parce qu'on a changé de test, hum